cmd/agent/app/integrations.go

```diff
@@ -192,23 +193,27 @@ func tuf(args []string) error {
 	}
 	args = append(args, implicitFlags...)
 
-	// Proxy support
-	proxies := config.GetProxies()
-	if proxies != nil {
-		proxyFlags := fmt.Sprintf("--proxy=%s", proxies.HTTPS)
-		args = append(args, proxyFlags)
-	}
-
 	// idx-flags go after the command and implicit flags
 	idxFlags, err := tufCmd.Flags().GetStringSlice("idx-flags")
 	if err == nil {
 		args = append(args, idxFlags...)
 	}
 
 	tufCmd := exec.Command(pipPath, args...)
+	tufCmd.Env = os.Environ()
```

Comment:
Just making sure: this inherits any other env var the user specifies at the command line, right?

Reply:
Yes